src/components/page/theming/ColorAccordion/styles.module.css

```diff
@@ -9,20 +9,20 @@
   list-style-type: none;
 
   /** primary **/
-  --ion-color-primary: #3880ff;
-  --ion-color-primary-rgb: 56, 128, 255;
+  --ion-color-primary: #0054e9;
+  --ion-color-primary-rgb: 0, 84, 233;
   --ion-color-primary-contrast: #ffffff;
   --ion-color-primary-contrast-rgb: 255, 255, 255;
-  --ion-color-primary-shade: #3171e0;
-  --ion-color-primary-tint: #4c8dff;
+  --ion-color-primary-shade: #004acd;
+  --ion-color-primary-tint: #1a65eb;
 
   /** secondary **/
-  --ion-color-secondary: #3dc2ff;
-  --ion-color-secondary-rgb: 61, 194, 255;
+  --ion-color-secondary: #0163aa;
+  --ion-color-secondary-rgb: 1, 99, 170;
   --ion-color-secondary-contrast: #ffffff;
   --ion-color-secondary-contrast-rgb: 255, 255, 255;
-  --ion-color-secondary-shade: #36abe0;
-  --ion-color-secondary-tint: #50c8ff;
+  --ion-color-secondary-shade: #015796;
+  --ion-color-secondary-tint: #1a73b3;
 
   /** tertiary **/
   --ion-color-tertiary: #5260ff;
```

Comment:
It looks like tertiary needs to change here https://ionic-docs-git-fw-5628-ionic1.vercel.app/docs/theming/basics